shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Control/Iterate/SetAlgebra.hs

```diff
@@ -308,7 +313,12 @@ testChain nm rep1 rep2 =
   testcase
     nm
     (ChainD (fromListD rep1 l4) (fromListD rep2 l5) (lift (\x (y, v) -> (x, y, v))))
-    [(1, (1, "m", 105)), (2, (2, "a", 101)), (6, (6, "b", 102)), (12, (12, "w", 108)), (34, (34, "a", 101)), (50, (50, "q", 107))]
+    [(1, (1, "m", 105)), (2, (2, "a", 101)), (6, (6, "b", 102)), (12, (12, "w", 108)), (50, (50, "q", 107))]
+
+-- L14 = [(1,"m"),(5,"z"),(6,"b"),(7,"r"),(12,"w"),(34,"v"),(50,"q"),(51,"l"),(105,"Z")]
+-- L15 = [("a",101),("b",102),("c",103),("f",104),("m",105),("q",107),("s",106),("w",108),("y",109),("zz",110)]
```

Comment:
can we delete this?

Reply:
more, no longer useful, debugging analysis.